fgspectra/models.py

```diff
@@ -1,89 +1,134 @@
-# -*- coding: utf-8 -*-
 r"""
```

Comment:
What about renaming this file `cross_spectra.py` or even just `cross.py`? It sounds more descriptive than `models` to me